src/pages/_app.tsx

```diff
@@ -1,11 +1,18 @@
 import "src/styles/globals.css"
-import type { AppProps } from "next/app"
+import type { AppProps } from "next/app";
+import { useState } from "react";
+import { CreatorContext } from "src/context";
+
 
 export default function App({ Component, pageProps }: AppProps) {
  
+  // const [ creator, setCreator] = useState('Full Send Podcast')
+const [creator, setCreator] = useState<string>("Full Send Podcast")
+const [payment, setPayment] = useState<string>("https://buy.stripe.com/test_9AQbLbdhD8Y5eYM3cf")
+const [image, setImage] = useState<string>("https://yt3.googleusercontent.com/5_TYQJ-59yU45NoK1GpQcRuov8OgZuwuSiS-0X8IveZI3QK_tKaiQxx9BHGqVkGogD08zY-txA=s900-c-k-c0x00ffffff-no-rj")
   return ( 
-       <>
+   <CreatorContext.Provider value = {{creator, setCreator, payment, setPayment, image, setImage}} >
```

Comment:
missing prop types

Reply:
The pageProps  are already typed with the AppProps type, which is imported from next/app.We don't know the exact type of pageProps because it can vary depending on the page being rendered.

Reply:
In the love of code I am on seeing any people